### DISCUSSION TITLE
Integration with simple-svelte-autocomplete

### DISCUSSION THREAD

**Discussion by bmccorm2** [skeletonlabs/skeleton#1372] [Q&A]
Similar to https://github.com/skeletonlabs/skeleton/issues/1317, I can't use the included Autocomplete component.  I have ~15,000 items in my list and the performance is not there.  I'd like to use https://www.npmjs.com/package/simple-svelte-autocomplete which supports asynchronous calls but I can't get the formatting to work.  

I'm relatively new but i'm not sure how to see which class is giving the issue.  Any help would be greatly appreciated!  My stackblitz: https://stackblitz.com/edit/skeletonlabs-repl-2gukdy?file=src/routes/+page.svelte

**Comment by endigo9740** [comment]
@bmccorm2 we'll be dropping an update within the next hour that removes the animations that were bottlenecking performance. 15k is still a LOT of items though, so I can't promise it'll be a great experience as is. I'm assuming `simple-svelte-autocomplete` does some sort of lazy loading, so we can certain look at that in the future.

However, I will also bring your attention to this upcoming change that will implement reactivity:
- https://github.com/skeletonlabs/skeleton/issues/1296

This may further enable this sort of a feature for you using our component. Truth be told I never expected folks to be throwing that may items at the autocomplete. Out of curiosity what's the use case here? What sort of data?

Also your Stackblitz doesn't seem to contain any usage of an autocomplete feature. It just say "Hi there".

![Screenshot 2023-04-25 at 11 15 12 AM](https://user-images.githubusercontent.com/1509726/234339017-d314a2fc-fda6-4e82-b7a2-f69383f9baa2.png)

**Reply by bmccorm2** [reply]
Sorry been having network issues today and must have copied that link before i posted it.  I updated the original post as well.

https://stackblitz.com/edit/skeletonlabs-repl-2gukdy?file=src/routes/+page.svelte

I tried modifying the component from the source code to remove the transitions but still wasn't able to use it :/  

I'm trying to build a finance app and want to be able to search for securities.  Some benchmarks have 20k+ securities and overall i was looking ~100k so that's why i was keen on `simple-svelte-autocomplete` which does the lazy loading.

BTW your project is so beautiful!! :)

**Reply by endigo9740** [reply]
No worries, I just wanted to let you know some progress is being made on this front. The new release of Skeleton dropped a couple hours ago and I would advise keeping an eye on these pending tickets. In the meantime we don't provide official support for third party libraries, but I will leave this open in case someone else has time to jump in an help!
